src/interaction/ArcBall.cpp

```diff
@@ -25,7 +25,7 @@
 ///\date 10/17/2019
 
 
-#include "ArcBall.h"
+#include "../../include/interaction/ArcBall.h"
```

Comment:
Are relative includes common?
With a CMake at the subfolder level, you can also have a 
`target_include_directories()` in the `CMakeList.txt` and that way you don't need relative includes

Reply:
See the comment below before this one

Reply:
indeed. Can we fix this changes on a new issue? 
I had to stop the cmake development in order to continue with the features and I have new code pointing to relative paths.